pkg/skaffold/errors/err_map.go

```diff
@@ -87,3 +87,20 @@ var knownBuildProblems = []problem{
 		},
 	},
 }
+
+// Deploy errors in deployment phase
+var knownDeployProblems = []problem{
+	{
+		regexp:  re("(?i).*Unable to connect.*"),
```

Comment:
you can change this expression to get the error details after Get  in the error message
```suggestion
		regexp:  re("(?i).*unable to connect to Kubernetes: Get (.*)"),
```